app/dev_mode/templates/dev-page.html

```diff
@@ -1,84 +1,180 @@
 {% extends 'layouts/_base.html' %} {% block page_title %}EQ Development{% endblock %} {% block content %}
 
+<!-- this page needs a total rethink, but this tidies things up a little for now -->
+
+<style media="screen">
+
+* { font-family: "lato" }
+
+label { display:inline-block; font-weight: 600; margin-bottom: 0.25rem; width:200px; }
+
+.field-container {
+  display:block;
+  margin: 0 0 1rem 0;
+  position: relative;
+}
+
+.field-container span {
+  position: relative;
+}
+
+input[type=text] {
+  padding:0.5rem;
+  width:200px;
+  font-size:1rem;
+}
+
+select {
+  width:200px;
+  font-size:1rem;
+  padding:0.55rem;
+  -webkit-appearance:none;
+  appearance:none;
+  border-radius:0;
+  border:1px solid #dfdfdf;
+  background:url(data:image/svg+xml;base64,PD94bWwgdmVyc2lvbj0iMS4wIiA/PjwhRE9DVFlQRSBzdmcgIFBVQkxJQyAnLS8vVzNDLy9EVEQgU1ZHIDEuMS8vRU4nICAnaHR0cDovL3d3dy53My5vcmcvR3JhcGhpY3MvU1ZHLzEuMS9EVEQvc3ZnMTEuZHRkJz48c3ZnIGVuYWJsZS1iYWNrZ3JvdW5kPSJuZXcgMCAwIDMyIDMyIiBoZWlnaHQ9IjMycHgiIGlkPSLQodC70L7QuV8xIiB2ZXJzaW9uPSIxLjEiIHZpZXdCb3g9IjAgMCAzMiAzMiIgd2lkdGg9IjMycHgiIHhtbDpzcGFjZT0icHJlc2VydmUiIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyIgeG1sbnM6eGxpbms9Imh0dHA6Ly93d3cudzMub3JnLzE5OTkveGxpbmsiPjxwYXRoIGQ9Ik0yNC4yODUsMTEuMjg0TDE2LDE5LjU3MWwtOC4yODUtOC4yODhjLTAuMzk1LTAuMzk1LTEuMDM0LTAuMzk1LTEuNDI5LDAgIGMtMC4zOTQsMC4zOTUtMC4zOTQsMS4wMzUsMCwxLjQzbDguOTk5LDkuMDAybDAsMGwwLDBjMC4zOTQsMC4zOTUsMS4wMzQsMC4zOTUsMS40MjgsMGw4Ljk5OS05LjAwMiAgYzAuMzk0LTAuMzk1LDAuMzk0LTEuMDM2LDAtMS40MzFDMjUuMzE5LDEwLjg4OSwyNC42NzksMTAuODg5LDI0LjI4NSwxMS4yODR6IiBmaWxsPSIjMTIxMzEzIiBpZD0iRXhwYW5kX01vcmUiLz48Zy8+PGcvPjxnLz48Zy8+PGcvPjxnLz48L3N2Zz4=);
+  background-position: center right 5px;
+  background-repeat:no-repeat;
+  background-size:22px;
+}
+
+.refresh {
+  width:32px;
+  height:auto;
+  display:block;
+  position:absolute;
+  top: -6px;
+  right: 4px;
+  cursor:pointer;
+}
+
+.qa-btn-submit-dev {
+  background:#6099c6;
+  color:white;
+  border:0;
+  border-radius:3px;
+  display:block;
+  padding:0.5rem 2rem;
+  text-align: center;
+  font-size: 1rem;
+}
+
+</style>
 
 <div class="container">
   <div class="panel u-mt-l">
     <div role="main" class="panel__body" id="main" tabindex="-1">
+
+      <h1>eQ Pre-production</h1>
```

Comment:
re widths - i didnt have the live data to see the changes - i'll extend that select out